butterknife-compiler/src/main/java/butterknife/compiler/ButterKnifeProcessor.java

```diff
@@ -257,7 +257,8 @@
 
     // Process each @BindView element.
     for (Element element : env.getElementsAnnotatedWith(BindView.class)) {
-      if (!SuperficialValidation.validateElement(element)) continue;
+      // we don't SuperficialValidation.validateElement(element)
```

Comment:
nit: capitalize W and end with a period. [Comment should be sentences][1]. Same for the next comment.

 [1]: nedbatchelder.com/blog/201401/comments_should_be_sentences.html